RIGImageGalleryTests/ImageGalleryViewControllerTests.swift

```diff
@@ -58,19 +58,19 @@ class ImageGalleryViewControllerTests: XCTestCase {
         // code coverage of the no delegate dismiss function
         imageGallery.dismissPhotoView(UIBarButtonItem())
         imageGallery.actionButton = UIBarButtonItem()
-        let actionFired = self.expectationWithDescription("action will fire on completion")
+        let actionFired = self.expectation(description: "action will fire on completion")
         imageGallery.actionButtonHandler = { _ in
             actionFired.fulfill()
         }
-        imageGallery.performSelector((imageGallery.navigationItem.rightBarButtonItem?.action)!, withObject: imageGallery)
-        waitForExpectationsWithTimeout(1.0, handler: nil)
+        imageGallery.perform((imageGallery.navigationItem.rightBarButtonItem?.action)!, with: imageGallery)
+        waitForExpectations(timeout: 1.0, handler: nil)
         imageGallery.doneButton = UIBarButtonItem()
-        let dismissFired = expectationWithDescription("dismiss handler will fire on completion")
+        let dismissFired = expectation(description: "dismiss handler will fire on completion")
         imageGallery.dismissHandler = { _ in
             dismissFired.fulfill()
         }
-        imageGallery.performSelector((imageGallery.navigationItem.leftBarButtonItem?.action)!, withObject: imageGallery)
-        waitForExpectationsWithTimeout(1.0, handler: nil)
+        imageGallery.perform((imageGallery.navigationItem.leftBarButtonItem?.action)!, with: imageGallery)
+        waitForExpectations(timeout: 1.0, handler: nil)
     }
 
     func testStatusBarHidden() {
```

Comment:
#req test fails to compile, remove the `()`  after `prefersStatusBarHidden` to fix